src/ports/postgres/modules/kmeans/kmeans.py_in

```diff
@@ -34,6 +37,25 @@ def kmeans_validate_src(schema_madlib, rel_source, **kwargs):
 
 # ----------------------------------------------------------------------
 
+def kmeans_validate_expr(schema_madlib, rel_source, expr_point, **kwargs):
+    if not columns_exist_in_table(rel_source, [expr_point]):
+
+        p = re.compile('[Aa][Rr][Rr][Aa][Yy]\s*\[\s*[\s*\w|,\s*]+\s*\]')
+
+        if p.match(expr_point.strip()):
+            view_name = unique_string('km_view')
+
+            plpy.execute(""" CREATE VIEW {view_name} AS
```

Comment:
Would this be better as a TEMP view? Or do all the callers create permanent tables?